Source/ACE.Server/Command/Handlers/DeveloperLootCommands.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+using ACE.DatLoader;
+using ACE.DatLoader.FileTypes;
+using ACE.Entity.Enum;
+using ACE.Server.Factories;
+using ACE.Server.Network;
+namespace ACE.Server.Command.Handlers
+{
+    public static class DeveloperLootCommands
+    {
+        [CommandHandler("testlootgen", AccessLevel.Admin, CommandHandlerFlag.ConsoleInvoke, 1, "Generates Loot for testing LootFactories", "<number of items> <loot tier>")]
+        public static void TestLootGenerator(Session session, params string[] parameters)
+        {
+            // This generates loot items and displays the drop rates of LootFactory
+
+            // Switch for different options
+            switch (parameters[0])
+            {
+                case "-info":
+                    Console.WriteLine("This is for more info on how to use this command");
+                    break;
+                default:
+                    break;
+            }
+
+            if (Int32.TryParse(parameters[0], out int numberItemsGenerate))
+            {
+                Console.WriteLine("Number of items to generate " + numberItemsGenerate);
+            }
+            else
+            {
+                Console.WriteLine("numbr of items is not an integer");
+                return;
+            }
+
+            if (Int32.TryParse(parameters[1], out int itemsTier))
```

Comment:
the following command throws an exception:

testlootgen 1

The 4th parameter to the 'CommandHandler' should be 2 instead of 1

This exception does not crash the server, as it is caught elsewhere

Whenever accessing a particular offset of an array, it should be ensured that the array is actually of that length... ie. for an array of length 100, it will have indices 0-99. If i try to access index 100 of the array, it will throw an IndexOutOfRangeException

Reply:
I'm always forgetting that array indexes start at zero...